app/javascript/src/components/Dashboard/Contacts/utils.jsx

```diff
@@ -1,11 +1,12 @@
 import React from "react";
 
+import dayjs from "dayjs";
 import { MenuVertical } from "neetoicons";
-import { Avatar, Typography } from "neetoui";
+import { Avatar, Typography, Dropdown } from "neetoui";
 
-import { formatTime } from "./utils";
+export const formatTime = dateTime => dayjs(dateTime).format("MMM, D, YYYY");
 
-export const CONTACTS_TABLE_COLUMN_DATA = [
+export const tableColumn = setShowDeleteAlert => [
```

Comment:
Find a better name for this utility function. This name doesn't resonate with what this function is actually returning.